### ISSUE TITLE
When removing Sublime Text, apt asks for confirmation

### ISSUE THREAD

**Issue by Crilum** [Botspot/pi-apps#1195]
I just updated Sublime Text, and apt asked for confirmation with the regular `Do you want to continue? [Y/n] ` when removing. Shouldn't it/install_packages use the `-y` flag?

**Comment by theofficialgman** [comment]
@Crilum you didn't look at the sublime text install scripts.... they very much do not use install_packages
```
#!/bin/bash
echo "Adding GPG key..."
wget -qO - https://download.sublimetext.com/sublimehq-pub.gpg | sudo apt-key add - || error "Failed to add GPG key for sublime-text repository!"
echo "deb https://download.sublimetext.com/ apt/stable/" | sudo tee /etc/apt/sources.list.d/sublime-text.list

sudo apt update || error "Failed to run 'sudo apt update'!"

sudo apt install sublime-text || error "APT failed to install sublime-text package!"

```

**Comment by theofficialgman** [comment]
it probably shouldn't use install_packages anyway since its just a single package from an apt repo
but it should have -y added to it for sure

**Comment by theofficialgman** [comment]
botspot seems to have implemented it how he wants it @Crilum 
https://github.com/Botspot/pi-apps/commit/e006f6f1591673a3f0c409a825a6954cdfcea5fe

**Comment by Botspot** [comment]
> botspot seems to have implemented it how he wants it @Crilum [e006f6f](https://github.com/Botspot/pi-apps/commit/e006f6f1591673a3f0c409a825a6954cdfcea5fe)

Yes, I forgot to reply back to this issue after making the change. (been busier with other things lately)
Commit https://github.com/Botspot/pi-apps/commit/e006f6f1591673a3f0c409a825a6954cdfcea5fe solves the issue.
@Crilum, thank you for letting us know.
